integrations/Make/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
**chore(blocking):**

Use the following tsconfig.json we possible, we are working on making all of them standard:
```json
{
  "extends": "../../tsconfig.json",
  "compilerOptions": {
    "target": "es2017",
    "baseUrl": ".",
    "outDir": "dist",
    "checkJs": false
  },
  "include": [".botpress/**/*", "src/**/*", "package.json"]
}
```